attest/verifier/build.rs

```diff
@@ -193,50 +183,13 @@ fn generate_sim_files(data_path: impl AsRef<Path>) {
         extensions: vec![dcap_extensions()],
     };
     let (signer_cert_pem, signer_key) = create_certificate(issuer, &subject);
-    write(chain_path, signer_cert_pem + &root_cert_pem).expect("Unable to write cert chain");
+    fs::write(chain_path, signer_cert_pem + &root_cert_pem).expect("Unable to write cert chain");
     write_signing_key(signer_key_path, &signer_key);
 
     generate_qe_identity(qe_identity_path, &validity, &signer_key);
     generate_tcb_info(tcb_info_path, &validity, &signer_key);
 }
 
-/// Returns true of the build script should generate the sim files
-fn should_generating_sim_files(data_path: impl AsRef<Path>) -> bool {
-    let data_path = data_path.as_ref();
-    cargo_emit::rerun_if_env_changed!("MC_SEED");
-
-    let generated_files = GENERATED_FILENAMES
-        .iter()
-        .map(|file| data_path.join(file))
-        .collect::<Vec<_>>();
-    for file in generated_files.iter() {
-        cargo_emit::rerun_if_changed!(file.to_str().expect("Could not stringify path"));
-    }
-
-    let regenerate_time = SystemTime::now()
-        - Duration::weeks(1)
-            .to_std()
-            .expect("Failed to convert to std duration");
-
-    if generated_files.iter().any(|file| {
-        file.metadata()
-            .and_then(|f| f.modified().map(|t| t < regenerate_time))
-            .unwrap_or(true)
-    }) {
-        return true;
-    }
```

Comment:
were running into enough issues with certificate contention causing a  developer to rebuild everything, it seems like IFF the developer has a 30 day old certificate on their machine having them rebuild every 30 days is still cheaper than daily or more for the previous behavior.